torchvision/models/feature_extraction.py

```diff
@@ -172,8 +175,19 @@ def _warn_graph_differences(train_tracer: NodePathTracer, eval_tracer: NodePathT
     warnings.warn(msg + suggestion_msg)
 
 
+def _get_leaf_modules_for_ops():
```

Comment:
Shall we add the typing info here? Something like:
```suggestion
def _get_leaf_modules_for_ops() -> List[type]:
```